client/src/customHooks/index.js

```diff
@@ -0,0 +1,19 @@
+import { useState, useContext } from 'react';
+import SnackBarContext from '../context/SnackBarContext';
+
+export const useFormInput = initialValue => {
+  const [value, setValue] = useState(initialValue);
+  function onChange(e) {
+    setValue(e.target.value);
+  }
+  return {
+    value,
+    onChange
+  };
+};
+
+export const useSnackBar = () => {
+  const { openSnackBar, closeSnackBar } = useContext(SnackBarContext);
```

Comment:
Unnecessary custom hook

Reply:
It's just a wrapper function, Through which it abstracts the snack bar's functionally at one place.